enclosure/diymore/README.md

```diff
@@ -0,0 +1,4 @@
+# Enclosure for ESP8266 by diymore
```

Comment:
*creo* que yunshan es el nombre más conocidos, hay mil vendors... no se como ponerle, ja
diymore,
AKA yunshan (https://owenduffy.net/blog/?p=10312 and https://ucexperiment.wordpress.com/2016/12/18/yunshan-esp8266-250v-15a-acdc-network-wifi-relay-module/),
AKA lysignal (https://www.amazon.co.uk/Lysignal-ESP8266-7-30V-Network-Module/dp/B073W8LMST)

Reply:
Si.. vi similares.. Puedo poner en el README los que vemos que son similares. En el link de Ali que puse explicitamente dice que la marca en diymore ... que se yo..

Reply:
sería genial que cuando la gente busque el que tiene, encuentre esto. no te calientes